js/attendeeContact.js

```diff
@@ -98,29 +99,29 @@ function getParameterByName(name, callback)
 // Increment the badge number and set the date/time stamp fields
 // Return true or false (Depending upon success.
 function incrementBadge() {
-    var numActiveBadgesFieldId = 7;
-    var pickupDateFieldIds = [ 9, 11, 13, 15 ];
-    var pickupTimeFieldIds = [ 10, 12, 14, 16 ];
 
     console.log("incrementBadge Received a request and is processing it now");
-    var checkFieldValue = document.getElementsByName('attendee.customDataList[' + numActiveBadgesFieldId + '].value')[0].value;
-    var numActive = parseInt(document.getElementsByName('attendee.customDataList[' + numActiveBadgesFieldId + '].value')[0].value);
-    var elNumberActiveBadges = document.getElementsByName('attendee.customDataList[' + numActiveBadgesFieldId + '].value')[0];
+
+    var checkFieldValue = $("label:contains('Number of Active Badges'):first").next().find("input").val();
+    var numActive = parseInt(checkFieldValue);
+    var elNumberActiveBadges = $("label:contains('Number of Active Badges'):first").next().find("input");

```

Comment:
Oh another one.


var elNumberActiveBadges = $("label:contains('Number of Active Badges'):first").next().find("input");
var checkFieldValue = elNumberActiveBadges.val();
var numActive = parseInt(checkFieldValue);

I would have to look more at the code but do you really need all 3 variables or is this just a way to get to numActive?